VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimTimeline/ClaimPhase.test.tsx

```diff
@@ -1,19 +1,14 @@
 import React from 'react'
-
-import { context, render, RenderAPI } from 'testUtils'
-import { act, ReactTestInstance } from 'react-test-renderer'
-import { claim } from '../../../claimData'
 import ClaimPhase from './ClaimPhase'
-import PhaseIndicator from './PhaseIndicator'
-import { TextView, VAButton, VAIcon } from 'components'
-import { Pressable } from 'react-native'
-import { waitFor } from '@testing-library/react-native'
 import { when } from 'jest-when'
 
+import { context, render } from 'testUtils'
+import { screen, fireEvent } from '@testing-library/react-native'
```

Comment:
Import can be grouped with the other node modules

Reply:
updated